packages/flutter_tools/lib/src/ios/fallback_discovery.dart

```diff
@@ -82,50 +82,50 @@ class FallbackDiscovery {
     }
 
     try {
-      final Uri result = await _mDnsObservatoryDiscovery.getObservatoryUri(
-        packageId,
-        device,
-        usesIpv6: usesIpv6,
-        hostVmservicePort: hostVmservicePort,
-      );
+      final Uri result = await _protocolDiscovery.uri;
       if (result != null) {
         UsageEvent(
           _kEventName,
-          'mdns-success',
+          'log-success',
```

Comment:
I renamed `fallback-success` and `fallback-failure` to `log-success` and `log-failure`.  Do we care about usage data continuity?  If so, I can change the names back.

Reply:
Not really, change away

Reply:
The events were only added to understand the severity of the issues around this that we were seeing. Please feel free to modify the events into whatever is useful for this new way of doing things, including adding new ones if needed.

Reply:
👍